pkg/process/types.go

```diff
@@ -20,20 +20,11 @@ import (
 	google_protobuf "github.com/containerd/containerd/protobuf/types"
 )
 
-// Mount holds filesystem mount configuration
-type Mount struct {
-	Type    string
-	Source  string
-	Target  string
-	Options []string
-}
-
 // CreateConfig hold task creation configuration
 type CreateConfig struct {
 	ID               string
 	Bundle           string
 	Runtime          string
-	Rootfs           []Mount
```

Comment:
As this is an exported field (and type); should we check if there's external consumers? A quick search showed me; https://github.com/google/gvisor/blob/2342c658c7c2ccdc1e23d642a5873c80297669de/pkg/shim/service.go#L486

Not sure if that _should_ use this field (but probably needs to be checked)

Reply:
I agree with @thaJeztah I'd keep this for 2.0 (in which we plan to remove v1 anyway).